test/spirv_report.spt

```diff
@@ -0,0 +1,43 @@
+; RUN: llvm-spirv %s -to-binary -o %t.spv
+; RUN: llvm-spirv --spirv-print-report %t.spv | FileCheck %s --check-prefix=CHECK-DAG
```

Comment:
Can we use --spirv-print-report with other options '-r' for example?

Thanks

Reply:
Adding of a new option is non-disruptive in the meaning that it mimics all similar options all around it, and can be as much combined with other options as similar existing options do.